src/main/scala/xiangshan/frontend/icache/IPrefetch.scala

```diff
@@ -210,10 +214,14 @@ class IPrefetchPipe(implicit p: Parameters) extends IPrefetchModule {
     )
   ))
   val s1_itlb_exception = VecInit((0 until PortNumber).map(i =>
-    ResultHoldBypass(
-      valid = tlb_valid_pulse(i),
-      init = 0.U(ExceptionType.width.W),
-      data = ExceptionType.fromTlbResp(fromITLB(i).bits)
+    Mux(
+      ExceptionType.hasException(s1_exceptionFromBackend),
+      s1_exceptionFromBackend,
+      ResultHoldBypass(
+        valid = tlb_valid_pulse(i),
+        init = 0.U(ExceptionType.width.W),
+        data = ExceptionType.fromTlbResp(fromITLB(i).bits)
+      )
```

Comment:
Don't modify s1_itlb_exception directly, for debugging purposes it needs to be the ResultHoldBypass of the iTLB result.

Also, it is recommended to expand `s1_exceptionFromBackend` to Vec(2), then use `ExceptionType.merge`, instead of a handwrittend `Mux`.